app/javascript/components/mentoring/Session.tsx

```diff
@@ -203,7 +203,7 @@ export const Session = (props: SessionProps): JSX.Element => {
                 </Tab.Panel>
                 <Tab.Panel id="guidance" context={TabsContext}>
                   <Guidance
-                    notes={notes}
+                    notes={guidance.exercise}
```

Comment:
This `Guidance` component should be updated to render both the exercise guidance and the track guidance (if present). So add an extra section to the accordion. The Track guidance section should go below the Exercise guidance section.